api/models.py

```diff
@@ -126,3 +141,26 @@ class TramitacaoEvent(models.Model):
 
     class Meta:
         ordering = ('sequencia',)
+
+
+class Progresso(models.Model):
+    casas = Choices('camara senado')
+    local_casa = models.CharField(
+        max_length=6, choices=casas.items(),
+        help_text='Casa desta proposição.',
+        null=True)
+
+    fase_global = models.TextField(blank=True)
+
+    local = models.TextField(blank=True)
+
+    data_inicio = models.DateField('Data de início', blank=True)
```

Comment:
Acho que para datas, ao contrário de para texto, se usa `null=True` ao invés de `blank=True` (ou os dois).
https://stackoverflow.com/questions/8609192/differentiate-null-true-blank-true-in-django